onnx/defs/nn/defs.cc

```diff
@@ -21,6 +21,20 @@ const char* auto_pad_doc =
     "SAME_UPPER or SAME_LOWER mean pad the input so that the output spatial size match the input."
     "In case of odd number add the extra padding at the end for SAME_UPPER and at the "
     "beginning for SAME_LOWER. VALID mean no padding.";
+const char* conv_auto_pad_14_doc =
```

Comment:
change name to conv_auto_pad_doc ... 14 is not required here... this file contains schema for the latest version of ops...